### PR TITLE
Organization membership

### PR DESCRIPTION
**Pull request by dmytro-lazebnyi**

### CODE REVIEW

File: scalecube-organization/src/test/java/io/scalecube/organization/UpdateOrganizationTest.java [hunk 1]
```diff
@@ -37,11 +37,11 @@ void updateOrganizationWithExistingOrgNameShouldFailWithNameAlreadyInUseExceptio
   }
 
   @Test
-  void updateOrganizationWithIdNotExistsShouldFailWithEntityNotFoundException() {
+  void updateOrganizationWithIdNotExistsShouldFailWithOrganizationNotFoundException() {
```
Comment (ronenhamias) [inline scalecube-organization/src/test/java/io/scalecube/organization/UpdateOrganizationTest.java]: i think we need to change the names of tests for example 

`shouldFailWithOrganizationNotFoundException`

the updateOrganizationWithIdNotExists feels very much redundant.

File: scalecube-organization-api/src/main/java/io/scalecube/account/api/Role.java [hunk 2]
```diff
@@ -1,17 +1,17 @@
 package io.scalecube.account.api;
 
 public enum Role {
-  Owner("owner"),
-  Admin("admin"),
-  Member("member");
+  Owner(300),
```
Comment (ronenhamias) [inline scalecube-organization-api/src/main/java/io/scalecube/account/api/Role.java]: not sure i like this ranking technique
maybe role and rank are different thing?

Comment (segabriel) [reply scalecube-organization-api/src/main/java/io/scalecube/account/api/Role.java]: @ronenhamias we have some vertical priority structure of these roles, so in the beginning this solution with ranks pretty well. It's better than just rely on the enum ordinal.
But ok maybe let's change this in another PR after some discussion and a concrete proposal.

Comment (ronenhamias) [reply scalecube-organization-api/src/main/java/io/scalecube/account/api/Role.java]: well ok i am just afraid this should be solved not as part of this lib

Comment (scooter72) [reply scalecube-organization-api/src/main/java/io/scalecube/account/api/Role.java]: flat role list is assumed in some systems roles are arranged in hierarchies

**Pull request merged**